### PR TITLE
Add plotting functionality

### LINKED ISSUE #8: Add plotting library
As I have discontinued `tastypl` support, I'm hoping to get charting up and running. Will look something like:

```
tw plot -y 5
```

This will generate a `.png` in the current directory containing the portfolio history over the past 5 years. Main options should be net liquidity and adjusted realized P/L (so including fees).

Time frame:
- All time
- 5 year
- 1 year
- YTD
- 6 month
- 1 month
- 5 day
- 1 day

Until we have the ability to get transaction history using the API, we may have to pass the `.csv` file as an additional argument.

### PR DESCRIPTION
**Pull request by Graeme22**
- work on plot module; add test data; clean up readme
- Add plotting functionality
- Realized P/L charts, net liquidity charts, current positions

Fixes #8.

### CODE REVIEW

File: src/controller.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        self.app.args.print_help()
+
+    @ex(
+        help='chart your portfolio\'s net liquidity or realized profit/loss over time',
+        arguments=[
+            (
+                ['csv'],
+                {
+                    'action': 'store',
+                    'help': 'csv file containing full portfolio transaction history'
```
Comment (def-mycroft) [inline src/controller.py]: I assume this is saying that the `csv` arg should be a path to the csv file, I recommend rewording the help text to specify that this should be a filepath string.

File: README.md [hunk 2]
```diff
@@ -1,43 +1,49 @@
...
 
-Currently unavailable pending release:
+## Installation
 
 ```
 $ pip install tastyworks-cli
 ```
 
-## Development
+## Usage
```
Comment (def-mycroft) [inline README.md:11]: users need more specific instruction on what csv file is required for the portfolio history functions e.g. `tw plot`. this might constitute a standalone section in the readme. 

it isn't necessarily obvious where this csv file should come from, so I recommend just stating "the csv file can be exported from the TW desktop app from the xyz tab", something of that nature. 

another thing - are there any other specifications on the csv file, i.e. what happens the file is exported with a starting date that splits a chain? 

so basically the readme just needs a description of the csv file required for `tw plot` input etc.

File: src/controller.py (hunk 1 above)
Comment (def-mycroft) [reply src/controller.py]: also see my comment on the readme, it wouldn't take very much to shed a little bit more light on the nature of the csv file

**Review by def-mycroft — CHANGES_REQUESTED** [review]
hi @Graeme22 , I have a couple of minor suggestions - 1) add a quick description of how the csv file is gotten and any other salient details and 2) change the help message in the controller to specify that a filepath string is required (currently it says 'csv file' which is ambiguous).

**Review by def-mycroft — APPROVED** [review]

**Comment by def-mycroft** [comment]
lgtm!! nice explanation on the README, saves people a ton of frustration.
